src/cpp/src/llm_pipeline.cpp

```diff
@@ -53,9 +53,28 @@ SchedulerConfig get_latency_oriented_scheduler_config() {
 }
 
 bool explicitly_requires_paged_attention(const ov::AnyMap& properties) {
-    return properties.find(ov::genai::scheduler_config.name()) != properties.end() ||
-           properties.find(utils::DRAFT_MODEL_ARG_NAME) != properties.end() ||
-           properties.find(ov::genai::prompt_lookup.name()) != properties.end();
+    if (properties.find(ov::genai::scheduler_config.name()) != properties.end()) {
+#ifdef OPENVINO_ARCH_X86_64
+        return true;
+#else
+        OPENVINO_THROW("Continuous batching backend requires PagedAttention operation support, which is available on x86_64 platform only");
+#endif
+    }
```

Comment:
OV CPU Plugin already supports PA on ARM with SVE and going to extend on NEON as well.
For better scalability does it makes sense to extend `ov::device::capabilities` property with the info if PA is supported?

Reply:
added a check that ARM64 with SVE is also supported.